### ISSUE TITLE
Starting clue tokens

### ISSUE THREAD

**Issue by LordPyrex** [NPBruce/valkyrie#547]
Is there a way to use the QItem starting items to create a banner at the bottom of the starting items page for quantity of starting clue tokens? Similar to how FFG implements. I'm currently handling this via dialog box, but figured it might be something of interest.

**Comment by redwolf2** [comment]
The current solution to it is to have it in the text of the first event.

**Comment by LordPyrex** [comment]
Yes, that's what I'm using now. I didn't know if there was a way to add it
in to the starting items screen in a future release.

**Comment by redwolf2** [comment]
Just a note: The text to it is SETUP_STARTING_CLUETOKEN in the localisation file.

**Comment by antontimmermans** [comment]
@redwolf2 your note suggests an alternative solution to "handing out Clue Tokens described in the start-up event". Do you have an example for this?

**Comment by redwolf2** [comment]
Try the text `{ff:SETUP_STARTING_CLUETOKEN}`

**Comment by antontimmermans** [comment]
I thought (hoped) that your suggestion would give a message in the FFG flow style: Select Investigators -> hand out items  AND Clues -> start the introduction. It does give a nice 'standard text' handing out zero (0) Clue tokens if you put it the dialogue box of an event:
![image](https://user-images.githubusercontent.com/42072372/53169017-6419cb00-35dc-11e9-9b77-2d22352b65eb.png)
Useful if you don't want to hand-out clue tokens :) ... but less if for example: you want to hand out 1 Clue with 3 or more investigators and 2 Clues if only 2 investigators are selected.
So if a mechanism to implement this in Valkyrie is to complicated/not needed then I would propose to close this issue and to keep solving this in the scenario.
I will raise a ticket in scrubbles's MOM Tutorial anyway to suggest that the first event triggered by EventStart should only hand-out the Clues, and that the next event will start with the introduction UI etc to mimic the std FFG scenario behavior.
(which reminds me.. I need to do this in my own scenario too :)

**Comment by redwolf2** [comment]
Try to use it like `{ffg:SETUP_STARTING_CLUETOKEN:{0}:123}`

To request 3 clue token.

**Comment by scrubbless** [comment]
Hi @redwolf2

Could you elaborate we a little bit?

Where should I put the 'Fox' and 'Dog' text (localisation file?) And how do I call them (Is it as simple as 

"Localisation:

0,Fox
1,Dog


Event1, The quick, brown {0} jumps over the lazy {1}."

Or do I need to use the {ffg:  } argument somehow?

**Comment by redwolf2** [comment]
Localisation:

```
MYTEXT,The quick, brown {0} jumps over the lazy {1}.
Event1,{qst:MYTEXT:{0}:Fox:{1}:Dog}
```

Or:

```
MYTEXT,The quick, brown {0} jumps over the lazy {1}.
DOG,Dog
FOX,Fox
Event1,{qst:MYTEXT:{0}:{qst:DOG}:{1}:{qst:FOX}}
```
